daml-assistant/integration-tests/src/DA/Daml/Assistant/CreateDamlAppTestsMain.hs

```diff
@@ -0,0 +1,11 @@
+-- Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+-- Wrapper around the tests so we can change the arguments
```

Comment:
Am I understanding this correctly that this enables caching of the compilation of the other main?

Reply:
Exactly